src/pages/developers/chains/functionality.mdx

```diff
@@ -1,40 +1,72 @@
-| Deposit/withdraw fungible token    | EVM     | ✅      | ✅      | ✅  | ✅       |
-| Deposit and call (native coin)     | EVM     | ✅      | ✅      | ✅  | ✅       |
-| Deposit and call (fungible token)  | EVM     | ✅      | ✅      | ✅  | ✅       |
-| Withdraw and call (native coin)    | EVM     | ✅      | ✅      | ✅  | ✅       |
-| Withdraw and call (fungible token) | EVM     | ✅      | ✅      | ✅  | ✅       |
-| Deposit ZETA                       | EVM     | ✅      | ✅      | ✅  | ❌       |
-| TSS Direct Deposits                | EVM     | ⚠️      | ⚠️      | ⚠️  | ❌       |
-| Withdraw ZETA                      | EVM     | ✅      | ✅      | ✅  | ❌       |
-| Deposit and call (ZETA)            | EVM     | ❌      | ❌      | ❌  | ❌       |
-| Withdraw and call (ZETA)           | EVM     | ❌      | ❌      | ❌  | ❌       |
-| Call                               | EVM     | ✅      | ✅      | ✅  | ✅       |
-| Deposit/withdraw native coin       | Bitcoin | ✅      | ✅      | ✅  | ❌       |
-| Deposit and call (native coin)     | Bitcoin | ✅      | ✅      | ✅  | ❌       |
-| Deposit/withdraw native coin       | Solana  | ✅      | ✅      | ✅  | ❌       |
-| Deposit/withdraw fungible token    | Solana  | ✅      | ✅      | ❌  | ❌       |
-| Deposit and call (native coin)     | Solana  | ✅      | ✅      | ✅  | ❌       |
-| Deposit and call (fungible token)  | Solana  | ✅      | ✅      | ❌  | ❌       |
-| Withdraw and call (fungible token) | Solana  | ❌      | Mar 10  | ✅  | ❌       |
-| Withdraw and call (native coin)    | Solana  | ❌      | Mar 10  | ✅  | ❌       |
-| Deposit ZETA                       | Solana  | ❌      | ❌      | ❌  | ❌       |
-| Withdraw ZETA                      | Solana  | ❌      | ❌      | ❌  | ❌       |
-| Deposit and call (ZETA)            | Solana  | ❌      | ❌      | ❌  | ❌       |
-| Withdraw and call (ZETA)           | Solana  | ❌      | ❌      | ❌  | ❌       |
-| Call                               | Solana  | ❌      | ❌      | ❌  | ❌       |
-| Deposit/withdraw native coin       | TON     | ❌      | ❌      | ✅  | ❌       |
-| Deposit/withdraw fungible token    | TON     | ❌      | ❌      | ❌  | ❌       |
-| Deposit and call (native coin)     | TON     | ❌      | ❌      | ✅  | ❌       |
-| Deposit and call (fungible token)  | TON     | ❌      | ❌      | ❌  | ❌       |
-| Withdraw and call (native coin)    | TON     | ❌      | ❌      | ❌  | ❌       |
-| Withdraw and call (fungible token) | TON     | ❌      | ❌      | ❌  | ❌       |
-| Deposit ZETA                       | TON     | ❌      | ❌      | ❌  | ❌       |
-| Withdraw ZETA                      | TON     | ❌      | ❌      | ❌  | ❌       |
-| Deposit and call (ZETA)            | TON     | ❌      | ❌      | ❌  | ❌       |
-| Call                               | TON     | ❌      | ❌      | ❌  | ❌       |
+## EVM
+
+| Feature                            | Chain | Mainnet | Testnet | E2E | Localnet |
```

Comment:
Why keeping chain as attribute if we have a table per chain (which I acutally think is more visual)

Reply:
1453ca5